packages/eslint-plugin/src/rules/no-visibility-modifiers.js

```diff
@@ -1,12 +1,7 @@
 // @ts-check
-const _internalApiMisc =
```

Comment:
in `@typescript-eslint/eslint-plugin` internal [API isn't accessible anymore like this](https://github.com/typescript-eslint/typescript-eslint/pull/6458)